extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosImporter.java

```diff
@@ -172,7 +175,9 @@ String importSingleAlbum(UUID jobId, TokensAndUrlAuthData authData, PhotoAlbum i
       throws IOException, InvalidTokenException, PermissionDeniedException {
     // Set up album
     GoogleAlbum googleAlbum = new GoogleAlbum();
-    String title = COPY_PREFIX + inputAlbum.getName();
+    String copyOf = getOrCreateStringDictionary(jobId).get(GoogleMultilingualString.CopyOf);
+    String title = MessageFormat.format(copyOf, Strings.nullToEmpty(inputAlbum.getName()));
```

Comment:
`MessageFormat.format` creates a new MessageFormat to do the formatting, so it should be thread safe